db/migrate/20230210103927_add_users_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :users, null: false, foreign_key: true
+  end
```

Comment:
- [ ] According to the [RD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) a `Post` should belong to a `:author` with `foreign_key` in the `users` table, such as:
```ruby 
 add_reference :posts, :author, null: false, foreign_key: { to_table: 'users' }
```